client/src/api/index.js

```diff
@@ -171,11 +171,12 @@ export const CbAdmin = {
       { headers: { Authorization: tkn } },
     ),
 
-  create: ({ orgName, category, email, password, passwordConfirm }) =>
+  create: ({ orgName, category, email, password, passwordConfirm, region }) =>
     axios.post('/api/cb/register', {
       orgName,
       email,
       category,
+      region,
```

Comment:
@astroash You know, we could have just reversed this change, and then #462 would have been unnecessary.

Reply:
Yes, I added this on purpose. It seemed counter-intuitive to collect information that wasn't being sent to the server. If it's currently not needed, then it can be ignored in the handler.